File: Arranging_Coins/arrange.js
/**
 * @param {number} n
 * @returns {number}
 */
var arrangeCoins = function (n) {
	let result = 0;

	for (let i = 1; i <= n; i++) {
		n -= i;
		result++;
		if (n < i) {
			break;
		}
	}

	return result;
};

File: Arranging_Coins/arrange.cs
public class Solution
{
    public int ArrangeCoins(int n)
    {
        int result = 0;

        for (int i = 1; i <= n; i++)
        {
            n -= i;
            result++;
            if (n < i)
            {
                break;
            }
        }

        return result;
    }
}
